Assignment5/Q1.py

```diff
@@ -0,0 +1,150 @@
+# coding=utf-8
+"""The solution presented below takes θ(N+M) runtime where N is the number of characters in dictionary words (sum of
```

Comment:
I meant big theta the upper and lower bound on the worst case running time. I think what I written is correct cause if the input dictionary has for example only words of length 1 then it gives the lower bound N+M. Is it better to use just big O then?